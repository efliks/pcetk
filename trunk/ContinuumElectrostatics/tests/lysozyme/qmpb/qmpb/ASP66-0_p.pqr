../0/site_ASP_66_p.pqr