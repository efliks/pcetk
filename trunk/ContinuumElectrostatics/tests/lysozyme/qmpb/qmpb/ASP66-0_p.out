Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named ASP66-0_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (10.02, 11.668, 27.1) 101 1
Finer level:    (10.02, 11.668, 27.1) 101 0.5
Finest level:   (10.02, 11.668, 27.1) 101 0.25


Self energy of ASP66-0_p in three dielectric environment = 155.019
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of ASP66-0_p with background charge set in three dielectric environment = -1.54978
(probably in kcal/mole)



Interaction energies of ASP66-0_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.00134668
0   1 : 4.71992e-06
1   0 : -1.14243e-05
1   1 : -0.000936193
2   0 : 0.000253764
2   1 : 5.47129e-06
3   0 : 0.00238314
3   1 : -9.97982e-05
3   2 : -1.76467e-05
3   3 : -0.00246085
4   0 : 0.000103779
4   1 : -0.000289249
5   0 : 0.00010208
5   1 : -0.00151531
6   0 : -5.64537e-05
6   1 : -0.000300854
7   0 : 0.000109801
7   1 : 3.79197e-06
8   0 : 0.000661068
8   1 : 0.000617784
9   0 : 0.00282374
9   1 : 0.0431699
10   0 : 0.000145235
10   1 : 0.0259957
11   0 : -0.515823
11   1 : 0.121159
12   0 : 310.038
12   1 : 393.298
13   0 : 0.000425711
13   1 : -0.00329725
14   0 : 0.0021359
14   1 : 3.70595e-05
15   0 : 0.00299773
15   1 : -1.04926e-06
16   0 : -0.000328308
16   1 : -0.00230467
17   0 : -0.000197
17   1 : 4.06624e-06
18   0 : 2.62482e-07
18   1 : -1.81346e-06
Total runtime: 7.75 s user 0.05 s system 
